homeassistant/components/dynalite/bridge.py

```diff
@@ -1,45 +1,32 @@
 """Code to handle a Dynalite bridge."""
 
+import asyncio
+
 from dynalite_devices_lib import DynaliteDevices
 from dynalite_lib import CONF_ALL
 
-from homeassistant.const import CONF_HOST
 from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_send
 
 from .const import DATA_CONFIGS, DOMAIN, LOGGER
-from .light import DynaliteLight
-
-
-class BridgeError(Exception):
-    """Class to throw exceptions from DynaliteBridge."""
-
-    def __init__(self, message):
-        """Initialize the exception."""
-        super().__init__()
-        self.message = message
 
 
 class DynaliteBridge:
     """Manages a single Dynalite bridge."""
 
-    def __init__(self, hass, config_entry):
+    def __init__(self, hass, host):
         """Initialize the system based on host parameter."""
-        self.config_entry = config_entry
         self.hass = hass
         self.area = {}
-        self.async_add_entities = None
-        self.waiting_entities = []
-        self.all_entities = {}
+        self.async_add_devices = None
+        self.waiting_devices = []
         self.config = None
```

Comment:
We don't need to store the bridge config as an instance attribute. Just store it in a local variable.

Reply:
done